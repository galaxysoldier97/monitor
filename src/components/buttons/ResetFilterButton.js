import React from 'react';
import { t } from 'mt-react-library/functions';
import { IconButton, Tooltip } from '@material-ui/core';
import AutorenewIcon from '@material-ui/icons/Autorenew';
import PropTypes from 'prop-types';

export const ResetFilterButton = ({onClick}) => {
  return (
    <Tooltip title={t('resetFilter')}><IconButton size="small" onClick={onClick}><AutorenewIcon /></IconButton></Tooltip>
  );
};

ResetFilterButton.propTypes = {
  onClick: PropTypes.func
};
