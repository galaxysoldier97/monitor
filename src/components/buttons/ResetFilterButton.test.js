import React from 'react';
import renderer from 'react-test-renderer';
import { ResetFilterButton } from './ResetFilterButton';

test('ResetFilterButton snapshot', () => {
  const component = renderer.create(<ResetFilterButton onClick={jest.fn} />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
