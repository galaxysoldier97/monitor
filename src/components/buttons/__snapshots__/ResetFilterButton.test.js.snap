// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResetFilterButton snapshot 1`] = `
<button
  aria-describedby={null}
  className="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeSmall"
  disabled={false}
  onBlur={[Function]}
  onClick={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseOver={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex={0}
  title={null}
  type="button"
>
  <span
    className="MuiIconButton-label"
  >
    <svg
      aria-hidden="true"
      className="MuiSvgIcon-root"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="M12 6v3l4-4-4-4v3c-4.42 0-8 3.58-8 8 0 1.57.46 3.03 1.24 4.26L6.7 14.8c-.45-.83-.7-1.79-.7-2.8 0-3.31 2.69-6 6-6zm6.76 1.74L17.3 9.2c.44.84.7 1.79.7 2.8 0 3.31-2.69 6-6 6v-3l-4 4 4 4v-3c4.42 0 8-3.58 8-8 0-1.57-.46-3.03-1.24-4.26z"
      />
    </svg>
  </span>
  <span
    className="MuiTouchRipple-root"
  />
</button>
`;
